litmus-portal/frontend/src/views/ChaosWorkflows/BrowseCluster/HeaderSection.tsx

```diff
@@ -104,25 +104,29 @@ const HeaderSection: React.FC<HeaderSectionProps> = ({
           focused
         >
           <InputLabel className={classes.selectText}>
-            {t('WorkflowCluster.header.status')}
+            {t('workflowCluster.header.status')}
           </InputLabel>
           <Select
             value={statusValue}
             onChange={changeStatus}
-            label="Workflow Status"
+            label="Cluster Status"
             className={classes.selectText}
           >
             <MenuItem value="All">
-              {t('WorkflowCluster.header.formControl.menu')}
+              {' '}
```

Comment:
If you can remove these spaces, it would be good. 
For ref: https://stackoverflow.com/questions/40817613/vscode-inserting-spaces-via-format-on-save